docs/interfaces/tensorboard.rst

```diff
@@ -1,21 +1,28 @@
 .. _tensorboards:
 
-##############
- TensorBoards
-##############
+###################
+ Using TensorBoard
+###################
 
-`TensorBoard <https://www.tensorflow.org/tensorboard>`__ is a widely used tool for visualizing and
-inspecting deep learning models. Determined makes it easy to use TensorBoard to examine a single
-experiment or to compare multiple experiments.
+`TensorBoard <https://www.tensorflow.org/tensorboard>`__ is a popular tool for visualizing and
+inspecting deep learning models. In Determined, you can use TensorBoard to for examining individual
+experiments or comparing multiple experiments.
```

Comment:
@MikhailKardash there is a grammar error / typo: please change the sentence to this:

`In Determined, you can use TensorBoard to examine individual experiments or compare multiple experiments.`